python/llm/README.md

```diff
@@ -106,7 +121,11 @@ You may run the models using `transformers`-style API in `bigdl-llm`.
 
   You may also convert Hugging Face *Transformers* models into native INT4 format for maximum performance as follows.
 
-  >**Note**: Currently only llama/bloom/gptneox/starcoder model family is supported; for other models, you may use the Transformers INT4 format as described above).
+  >**Notes**: 
+  
+  * Currently only llama/bloom/gptneox/starcoder/chatglm model family is supported; for other models, you may use the Transformers INT4 format as described above).
+  
+  * We have developed the `transformers`-style API for specific native models, you may choose the corresponding API to load the converted model.
```

Comment:
I don't think we need to call this transformers-style API. Just remove this sentence?